muikku-core-plugins/src/main/java/fi/otavanopisto/muikku/plugins/transcriptofrecords/TranscriptofRecordsPermissions.java

```diff
@@ -22,6 +22,15 @@
   @Scope (PermissionScope.ENVIRONMENT)
   @DefaultEnvironmentPermissionRoles ( { EnvironmentRoleArchetype.ADMINISTRATOR, EnvironmentRoleArchetype.MANAGER } )
   public static final String TRANSCRIPT_OF_RECORDS_FILE_UPLOAD = "TRANSCRIPT_OF_RECORDS_FILE_UPLOAD";
+  
+  @Scope (PermissionScope.ENVIRONMENT)
+  @DefaultEnvironmentPermissionRoles ( {
+    EnvironmentRoleArchetype.STUDY_GUIDER,
+    EnvironmentRoleArchetype.TEACHER,
```

Comment:
Teachers can access the student information in guider view, so they should be able to access their studies matrix too.